GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -651,6 +655,15 @@ private void InitializeComponent()
             this.ShowFirstParent.Size = new System.Drawing.Size(23, 20);
             this.ShowFirstParent.ToolTipText = "Show first parents";
             // 
+            // ShowLocks
+            // 
+            this.ShowLocks.DisplayStyle = System.Windows.Forms.ToolStripItemDisplayStyle.Image;
+            this.ShowLocks.Image = ((System.Drawing.Image)(resources.GetObject("ShowLocks.Image")));
+            this.ShowLocks.ImageTransparentColor = System.Drawing.Color.Magenta;
+            this.ShowLocks.Name = "ShowLocks";
+            this.ShowLocks.Size = new System.Drawing.Size(23, 20);
+            this.ShowLocks.ToolTipText = "Show locks";
```

Comment:
I prefer `Show LFS locks`.

Also, that can't be only displayed in the toolbar.
Perhaps add it also in the `repository` menu next to `Worktrees`

![image](https://user-images.githubusercontent.com/460196/88381296-720de580-cda6-11ea-9140-855da01da67a.png)